api/get_transaction_test.go

```diff
@@ -67,6 +67,14 @@ func (s *GetTransactionTestSuite) SetupTest() {
 	})
 	s.NoError(err)
 
+	_, err = s.storage.StateTree.Set(2, &models.UserState{
+		PubKeyID: 123,
+		TokenID:  models.MakeUint256(1),
+		Balance:  models.MakeUint256(0),
+		Nonce:    models.MakeUint256(0),
+	})
+	s.NoError(err)
```

Comment:
The API did not used to require that the receiver already existed. If the receiver did not exist by the time the txn was processed then it would fail. However, transactions are no longer allowed to fail! So the API now checks for existence and refuses the txn.